app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ingredients_analysis/adapter/IngredientAnalysisViewHolder.java

```diff
@@ -0,0 +1,15 @@
+package openfoodfacts.github.scrachx.openfood.views.product.ingredients_analysis.adapter;
```

Comment:
You need create a separate class for this. This can be added in the `...penfood/views/product/ingredients_analysis/adapter/IngredientAnalysisRecyclerAdapter.java` itself as a subclass. You can refer to other recycler view examples in the app for reference.

Reply:
okay! Actually I saw that CategoryListRecyclerAdapter and it's ViewHolder are seperate, so I followed the same pattern.